sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/ReactorNettyClient.java

```diff
@@ -306,6 +303,15 @@ private static ConnectionObserver getConnectionObserver() {
                     }
                     requestRecord.setTimeReceived(time);
                 }
+            } else if (state.equals(HttpClientState.DISCONNECTING)
+                || state.equals(HttpClientState.REQUEST_PREPARED)
```

Comment:
what is the difference between RESPONSE_COMPLETE vs RESPONSE_RECEIVED?

Reply:
HttpClientState.REQUEST_PREPARED is being handled above, maybe remove from here?

Reply:
I just wanted to reduce the noise level of logs when debug logs were enabled - because I needed debug logs for soem perf investigations.

Reply:
And I wanted to be able to double-check what other states we would see in the perf tests (when debug logs are enabled)